deploy/.env.example

```
@@ -0,0 +1,39 @@
+# Replace with your Docker Hub username/repo
+BACKEND_IMAGE=<dockerhub-id>/gachi-be:latest
+AI_IMAGE=<dockerhub-id>/gachi-ai:latest
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The AI_IMAGE key should go before the BACKEND_IMAGE key

(UnorderedKey)